Update repository to reflect connect-es rename (#366)
bufbuild/connect-es-integration: README.md

# Connect-ES Integration

> Connect-ES Integration is a repository of example projects using Connect-ES with various JS frameworks and tooling.

Reviewer: ```suggestion
Connect-ES Integration is a repository of example projects using Connect with various TypeScript web frameworks and tooling.
```

> It provides numerous examples for integrating Connect-ES into a project.

Reviewer: ```suggestion
It provides numerous examples for integrating Connect into a project.
```

> All projects test Connect-ES against the Eliza demo running at https://demo.connect.build.  Each project illustrates an example

Reviewer: ```suggestion
All projects test Connect against the Eliza demo running at https://demo.connect.build.  Each project illustrates an example
```
